src/uct/ib/base/ib_iface.c

```diff
@@ -1534,6 +1579,14 @@ ucs_status_t uct_ib_iface_query(uct_ib_iface_t *iface, size_t xport_hdr_len,
     iface_attr->latency.c += numa_latency;
     iface_attr->latency.m  = 0;
 
+    status = uct_ib_iface_get_cuda_latency(iface, &cuda_latency);
```

Comment:
@Akshay-Venkatesh can we add some flag in ib_iface to bypass this function? just in case something goes wrong
UCX_IB_CUDA_LATENCY_ENABLE=[y|n]